.github/workflows/cmake.yml

```diff
@@ -33,6 +33,21 @@ jobs:
         path: |
           build/libutf8proc.*
           build/Debug/utf8proc.*
+    - name: Test Consuming (Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: |
+        cmake --install build --prefix tmp/install --config Debug
```

Comment:
How about adding `-DCMAKE_INSTALL_PREFIX=tmp/install` to `cmake -S . ...` in the `name: Build` step instead of using `--prefix tmp/install` here?